Week1/homework/app.js

```diff
@@ -1,11 +1,135 @@
+  */
+
+  const bookObject = {
+    'Java Script': {
+      title: 'The Good Parts',
+      language: 'English',
+      author: 'Douglas Rockford',
+      source:
+        'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1328834793i/2998152.jpg',
+    },
+    'Eloquent JavaScript': {
+      title: 'A Modern Introduction to Programming',
+      language: 'English',
+      author: ' Marian Haversine',
+      source:
+        'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1308260856i/8910666.jpg',
+    },
+    'The JavaScript': {
+      title: 'The Definitive Guide',
+      language: 'French',
+      author: 'David Flanagan',
+      source:
+        'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1347652925i/148050.jpg',
+    },
+    'High Performance JavaScript': {
+      title: 'High Performance JavaScript',
+      language: 'Italian',
+      author: 'Nicholas C. Kakas',
+      source:
+        'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1328834985i/7008656.jpg',
+    },
+    'JavaScript Patterns': {
+      title: 'JavaScript Patterns',
+      language: 'German',
+      author: 'Stomas Stefanie',
+      source:
+        'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1344668503i/9422683.jpg',
+    },
+    'Secrets of the JavaScript Ninja': {
+      title: 'Secrets of the JavaScript Ninja',
+      language: 'French',
+      author: 'John Resign',
+      source:
+        'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1361224545i/4373732.jpg',
+    },
+    'Effective JavaScript': {
+      title: '68 Specific Ways to Harness the Power of JavaScript',
+      language: 'Italian',
+      author: 'David Herman',
+      source:
+        'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1372042097i/13705402.jpg',
+    },
+    'Professional JavaScript for Web Developers': {
+      title: 'Professional JavaScript for Web Developers',
+      language: 'German',
+      author: 'Nicholas C. Takas',
+      source:
+        'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1172064302i/135437._SX318_.jpg',
+    },
+    'Learning Javascript Design Patterns': {
+      title: 'Learning Javascript Design Patterns',
+      language: 'English',
+      author: 'Addr Osmanli',
+      source:
+        'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1344734794i/14289134.jpg',
+    },
+    Closure: {
+      title: 'The Definitive Guide',
+      language: 'Italian',
+      author: 'Michael Colin',
+      source:
+        'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1328761606i/8300295.jpg',
+    },
+  };
+
+  const createBookCard = () => {
+    /* Creating <ul> </ul> in the index.html */
+    const newUlElement = document.createElement('ul');
+    /* setting attribute to the <ul> </ul> */
+    newUlElement.setAttribute('id', 'grid-flex');
```

Comment:
`ID` attributes usually should be chosen in such a way that they can uniquely identify an element in a page. I think in this case, `grid-flex` seems to be a very generic selector which would be better suited as a class name rather than `id` selector. 🙂